item-spec/json-schema/item.json

```diff
@@ -79,6 +79,7 @@
                       "sar",
                       "sat",
                       "scientific",
+                      "tiles",
```

Comment:
The value to use here (and probably also in all other examples) is the identifier/folder name.

Reply:
👍